### DISCUSSION TITLE
graphql을 넣고 목록은 100개까지안뜬느데 실제로는 100개이상 적용되나요?

### DISCUSSION THREAD

**Discussion by kikikikikinanana** [changdoc/naver-vaccine-macro#174] [Q&A]
병원 200군데짜리 graphql을 넣어도 목록은 100개까지밖에안뜬느데 실제로는 200개 적용될까요?

**Comment by changdoc** [comment]
따로 제한을 두진 않았는데 확인해보도록 하겠습니다

**Reply by changdoc** [reply]
범위안에 들어오는 모든 병원이 노출되는걸로 기억했는데 지금 보니 데이터는 100개 이상 내려오지가 않네요. total에는 영역 안에 존재하는 수가 있긴한데 병원 정보는 100개까지로 제한이 걸려 있는 것으로 보입니다.
<img width="1701" alt="스크린샷 2021-08-02 오전 12 20 10" src="https://user-images.githubusercontent.com/1879494/127776270-047290ae-55b6-47e2-81a6-10e8ab686177.png">

**Reply by kikikikikinanana** [reply]
그렇군요 감사합니다
